[32mINFO    [0m integration.test_tls:test_tls.py:218 Removing relation
[32mINFO    [0m pytest_operator.plugin:plugin.py:790 Model status:

Model  Controller          Cloud/Region        Version  SLA          Timestamp
test   microk8s-localhost  microk8s/localhost  3.5.4    unsupported  01:36:00Z

App                       Version                  Status  Scale  Charm                     Channel        Rev  Address         Exposed  Message
mysql-k8s                 8.0.37-0ubuntu0.22.04.3  active      3  mysql-k8s                                  0  10.152.183.112  no       
self-signed-certificates                           active      1  self-signed-certificates  latest/stable  155  10.152.183.88   no       

Unit                         Workload  Agent      Address     Ports  Message
mysql-k8s/0*                 active    executing  10.1.96.11         
mysql-k8s/1                  active    executing  10.1.96.12         
mysql-k8s/2                  active    executing  10.1.96.13         
self-signed-certificates/0*  active    idle       10.1.96.14         

[32mINFO    [0m pytest_operator.plugin:plugin.py:796 Juju error logs:

unit-mysql-k8s-0: 01:34:55 ERROR unit.mysql-k8s/0.juju-log certificates:3: An unknown certificate expiring.
unit-mysql-k8s-1: 01:34:59 ERROR unit.mysql-k8s/1.juju-log certificates:3: An unknown certificate expiring.
unit-mysql-k8s-0: 01:35:01 ERROR unit.mysql-k8s/0.juju-log certificates:3: An unknown certificate expiring.
unit-mysql-k8s-2: 01:35:03 ERROR unit.mysql-k8s/2.juju-log certificates:3: An unknown certificate expiring.
unit-mysql-k8s-1: 01:35:08 ERROR unit.mysql-k8s/1.juju-log certificates:3: An unknown certificate expiring.
unit-mysql-k8s-0: 01:35:09 ERROR unit.mysql-k8s/0.juju-log certificates:3: An unknown certificate expiring.
unit-mysql-k8s-1: 01:35:35 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: relation scope id "d8aa7aed-d2ce-4d3d-8287-c7b6a7a9a776:r#3#self-signed-certificates": settings d8aa7aed-d2ce-4d3d-8287-c7b6a7a9a776:r#3#self-signed-certificates not found (not found)

[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Forgetting main...